server/services/store/sqlstore/user.go

```diff
@@ -27,26 +29,54 @@ func (s *SQLStore) GetRegisteredUserCount() (int, error) {
 }
 
 func (s *SQLStore) getUserByCondition(condition sq.Eq) (*model.User, error) {
+	users, err := s.getUsersByCondition(condition)
+	if err != nil {
+		return nil, err
+	}
+
+	var user *model.User
+
+	if len(users) > 0 {
```

Comment:
I think this can be clearer if you do something like:
```go
if len(users) == 0 {
    return nil, nil
}
return users[0], nil
```

Reply:
Done.